html/datasets.html

```diff
@@ -0,0 +1,331 @@
+                    {'field': 'publicationYear', 'display': 'Publication Year'},
+                    {'field': 'authors.name.exact', 'size': 20, 'display': 'Author'},
+                    {'field': 'authors.organization.name.exact', 'display': 'Author Organization'},
+                    {'field': 'dcoCommunities.name.exact', 'display': 'DCO Community'},
+                    {'field': 'portalGroups.name.exact', 'display': 'Portal Group'},
+                    {'field': 'projects.name.exact', 'display': 'Project'},
+                    {'field': 'dataTypes.name.exact', 'display': 'Data Type'}
+                ],
+                search_sortby: [
+                    {'display':'Year','field':'publicationYear', 'order':'asc'},
+                    {'display':'Title','field':'title.exact', 'order':'desc'}
+                ],
+                render_result_record: function(options, record)
+                {
+                /*
+                 Contents to be displayed in individual records.
+                 Here is a template below. (Not real)
+                 ****************************************************************************
+                 * (The title; with link if dataset has DOI) (Publication Year)
+                 * Type: Dataset
+                 * DCO-ID: (e.g.) 11121/8158-6700-3912-5989-CC (linked to metadata in vivo) (disabled)
+                 * DCO Communites: Data Science Team; Extreme Physics and Chemistry Community; ...
+                 * DCO Portal Groups: Data Science Community; ...
+                 * Authors: Ma, Xiaogang (Marshall); Ghiorso, Mark S.; Zhong, Hao; ...
+                 * Projects: DECADE - CO2 Soil Degassing at Piton de la Fournaise; ...
+                 * Has Data Types: Thermodynamics of chemicals and minerals; ...
+                 * Citations (Source Papers): Inverted High-Temperature Quartz, Unit Cell Parameters ...
+                 * Distributions: (Note: Some datasets have multiple distributions; some distributions have multiple files.)
+                 *     20150827-0138 (Direct access: GhiorsoEtAl1979)
+                 *     Han's Queries (Oct 2013) (Direct access: EPC Queries; DS Queries; DL Queries; DE Queries; README; RF Queries)
+                 *     ...
+                 * [Badges of doi and metadata with link][]..
+                 ****************************************************************************
+                 Below are implementations of each item.
+                */
+                    var doiUrl = "https://dx.doi.org/"+record["doi"];
+
+                    var html = "<tr><td>";
+
+                    // Show DCO logo as datasets are all DCO publications
+                    html += "<img style='vertical-align: middle' src='//deepcarbon.net/sites/default/files/images/dco-icon-footer_0.png' height='15' width='15'>&nbsp;";
+
+                    // display title; if the dataset has doi, display title with link
```

Comment:
check for record["uri"] first and link to https://deepcarbon.net/dco_dataset_summary?uri=that_record_uri

else check for the doi and if it exists link to it

else don't give a link